build/toolchain/android/BUILD.gn

```diff
@@ -60,14 +60,13 @@ template("android_gcc_toolchain") {
 
       cc = compiler_prefix + prefix + "/clang"
       cxx = compiler_prefix + prefix + "/clang++"
+      ar = prefix + "ar"
```

Comment:
this should be `prefix + "/ar"`

Reply:
Fixed that - I commited before saving the file again :)